### PR TITLE
Update v1beta2 -> v1.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

File: pgp/provenance/README.md [hunk 1]
```diff
@@ -7,12 +7,14 @@ Variant Data
...
 * Google Cloud Storage gs://pgp-harvard-data-public/ which is a subset of https://my.pgp-hms.org/public_genetic_data?utf8=%E2%9C%93&data_type=Complete+Genomics&commit=Search
 * _Note that one participant has been sequenced twice:_
  1. gs://pgp-harvard-data-public/hu34D5B9/GS000012763-DID/
  1. gs://pgp-harvard-data-public/hu34D5B9/GS000018040-DID/
 
 Description:
+
 * http://www.completegenomics.com/FAQs/
 * http://www.completegenomics.com/FAQs/Working-with-the-Data/#q1
 * http://www.completegenomics.com/customer-support/documentation/DataFileFormats-100357139.html
```
Comment (pgrosu) [inline pgp/provenance/README.md]: Not sure if these links still point to the right locations :(

Comment (mbookman) [inline pgp/provenance/README.md]: * http://www.completegenomics.com/customer-support/faqs/
* http://www.completegenomics.com/customer-support/documentation/100357139-2/

There is a "Working with the Data" drop-down on the FAQ page.

**Comment by mbookman** [comment]
LGTM

File: pgp/provenance/README.md (hunk 1 above)
Comment (pgrosu) [inline pgp/provenance/README.md]: Nice, thanks Matt!

**Pull request merged**